File: src/com/wt/pkg/Member.java
package com.wt.pkg;

public class Member {
	String firstname;
	String lastname;
	double age;
	int maxWeight;
	int minWeight;
	
	public int getMaxWeight() {
		return maxWeight;
	}
	public void setMaxWeight(int maxWeight) {
		this.maxWeight = maxWeight;
	}
	public int getMinWeight() {
		return minWeight;
	}
	public void setMinWeight(int minWeight) {
		this.minWeight = minWeight;
	}
	public String getFirstname() {
		return firstname;
	}
	public void setFirstname(String firstname) {
		this.firstname = firstname;
	}
	public String getLastname() {
		return lastname;
	}
	public void setLastname(String lastname) {
		this.lastname = lastname;
	}
	public double getAge() {
		return age;
	}
	public void setAge(double age) {
		this.age = age;
	}
}

File: Web/js/app/member.js
define([], function () {
	console.log("member!")
	
	function getHeader() {
		$.get( "MemberInfo", function( members ) {
			for(var member in members) {
				console.log(members[member].firstname);
				console.log(members[member].lastname);
				console.log(members[member].age);		
				
				$("#weightTrackerNav ul").append("<li><a href='#"+members[member].firstname+"'>"+members[member].firstname+"</a></li>");
			}
		  
		});
	}
	
	return {
		getHeader:getHeader
	};
});
